catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -608,12 +603,13 @@ class PoppySegmentedCommand(SegmentedAperture):
                       of the segmented DM being used
     :attribute list of coefficients: list of piston, tip, tilt coefficients in units of m, rad, rad
     """
-    def __init__(self, global_coefficients, dm_config_id, wavelength, rotation=0):
-        # Initilize parent class
-        super().__init__(dm_config_id=dm_config_id, wavelength=wavelength, rotation=rotation)
+    @poppy.utils.quantity_input(display_wavelength=u.nm)   # decorator provides a check on input units
+    def __init__(self, global_coefficients, dm_config_id, display_wavelength=640*u.nm, rotation=0, **kwargs):
```

Comment:
since the wavelength isn't used for the display in this class, it seems odd that it is called the `display_wavelenth`. You might want to think of a more general variable name.